lib/manageiq/appliance_console/cli.rb

```diff
@@ -256,6 +256,8 @@ def create_key
       unless key_configuration.activate
         raise "Could not create encryption key (v2_key)"
       end
+    rescue RuntimeError => e
```

Comment:
This looks like it would make the cli script return a success status to the command line after an error. This is not what we want.

I think a better solution would be to `exit` from the cli after writing the message above. Then we would get the message, exit with an error status, and not catch exceptions which are not this particular error situation (activate returning false).